spec/ios_get_build_number_spec.rb

```diff
@@ -13,14 +13,14 @@
       expect_build_number(xcconfig_mock_content: xcconfig_mock_content, expected_build_number: '1940')
     end
 
-    it 'parses an xcconfig file with keys with spaces and returns a nil build number' do
+    it 'parses an xcconfig file with keys with spaces and returns the correct build number' do
```

Comment:
This test was verifying something else given we didn't use xcodeproj for parsing the `.xcconfig` file (see [this comment](https://github.com/wordpress-mobile/release-toolkit/pull/458#discussion_r1178582147) from @spencertransier).

Reply:
Awesome, thanks for updating this test @iangmaia!